woocommerce-gutenberg-products-block.php

```diff
@@ -18,7 +18,72 @@
 
 defined( 'ABSPATH' ) || exit;
 
-if ( version_compare( PHP_VERSION, '5.6.0', '<' ) ) {
+define( 'WOOCOMMERCE_BLOCKS__MINIMUM_WP_VERSION', '5.0' );
+define( 'WOOCOMMERCE_BLOCKS__MINIMUM_PHP_VERSION', '5.6' );
+
+if ( version_compare( $GLOBALS['wp_version'], WOOCOMMERCE_BLOCKS__MINIMUM_WP_VERSION, '<' ) ) {
+	if ( defined( 'WP_DEBUG' ) && WP_DEBUG ) {
+		error_log( // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_error_log
```

Comment:
Why do we need error logs for this? I agree with admin notice but I don't think we should clutter up logs if this is not running, as it would trigger every page load for all users..

Reply:
Removed in 5f86e63d3668c6b38c1169fd6d4ca3fa85c77b2b.